src/ConfigurationProcessor.php

```diff
@@ -62,21 +75,41 @@ public function processInstallation(array $configuration)
         }
 
         $nativeModules = $configuration['modules'];
-        $processManager = new ProcessManager(100, 8);
+        $timeout = array_key_exists('timeout', $configuration) ?
+            $configuration['timeout'] :
+            self::WAIT_FOR_PROCESS_STATUS_TIMEOUT
+        ;
 
-        foreach ($nativeModules as $moduleName => $moduleVersion) {
-            $this->io->write(sprintf('<info>Looked into "%s" module (version %s)</info>', $moduleName, $moduleVersion));
+        $processes = array_key_exists('processes', $configuration) ?
+            $configuration['processes'] :
+            self::PARALLEL_PROCESSES
+        ;
 
+        $processManager = new ProcessManager($timeout, $processes);
+        $shouldOverWrite = $this->io->askConfirmation('Do you want to overwrite the existing modules? (y/n)');
```

Comment:
> does it mean you need to answer interactively all the time?

No, the default value is true if you execute `composer install --no-interaction`.

> maybe a parameter to set the overwrite would be nice as well: `--should-overwrite=true|false`

Yeah, maybe but the more configurable we make this plugin, the less coverage we have and so one the more we increase the risks of regression.

I really don't **want** to invest time on this plugin, so if you think it's important you may contribute to it.

Thanks for your understanding :+1:

Reply:
The `overwrite` parameter is a nice to have (although I see many use cases where it would be needed) and can be added later
However the `--no-interaction` is a MUST HAVE, for travis only it will be necessary (and for any automated deployment as well, I don't know how ready works?)

I looked into https://getcomposer.org/apidoc/1.5.6/Composer/IO/IOInterface.html#method_askConfirmation It's great if it manages the option automatically!